js/app.js

```diff
@@ -0,0 +1,91 @@
+// sign-in input validator
```

Comment:
To na przyszłość ale poczytaj o różnych wzorach w programowaniu. Podrzucam Ci przykładowo `modular pattern`:

https://medium.com/@tkssharma/javascript-module-pattern-b4b5012ada9f

Warto, żebyś to w miarę niedługo zaczął stosować, bo raczej unika się wystawiania całego kodu w global scopie